examples/Servo-Stop/Servo-Stop.ino

```diff
@@ -0,0 +1,36 @@
+#include <Servo.h>
+#define trigPin  4
+#define echoPin  2
+
+Servo myservo;  // create servo object to control a servo
+int pos = 0; // variable to store the servo position
+void setup() {
+  // put your setup code here, to run once:
```

Comment:
```suggestion
```
I don't think this comment is needed in every example. The user should already know this from studying the BareMinimum example.

Reply:
PR Updated